test/openssl/test_ssl.rb

```diff
@@ -1817,6 +1817,18 @@ def test_fileno
     sock2.close
   end
 
+  def test_export_keying_material
+    start_server do |port|
+      cli_ctx = OpenSSL::SSL::SSLContext.new
+      server_connect(port, cli_ctx) do |ssl|
+        assert_instance_of(String, ssl.export_keying_material('ttls keying material', 64))
+        assert_operator(64, :==, ssl.export_keying_material('ttls keying material', 64).b.length)
+        assert_operator(8, :==, ssl.export_keying_material('ttls keying material', 8).b.length)
+        assert_operator(5, :==, ssl.export_keying_material('test', 5, 'context').b.length)
+      end
```

Comment:
CI is failing on Windows because the client closes the socket too quickly and the server fails to finish TLS 1.3 handshake. (IIRC it used to fail on Linux too. Something must have changed in OpenSSL...)

I don't like this very much but the current workaround is to let it transmit some actual payload so that it can finish the handshake, by adding `ssl.puts "abc"; ssl.gets`.